shared/go.mod

```diff
@@ -3,35 +3,13 @@ module github.com/chanzuckerberg/happy/shared
 go 1.19
 
 require (
-	github.com/chanzuckerberg/go-misc v0.0.0-20221117151707-47f8234dc20c
```

Comment:
this was bumping the version since the CLI depends on this `shared` module

Reply:
figured it out by using `go mod graph | grep keyring` to see what other people in the package were using that go-misc or keyring library